### PR TITLE
Add libvirt_pool type with virsh provider

### PR DESCRIPTION
**Pull request by maxenced**
First version to provide #7  , not tested yet

### CODE REVIEW

**Comment by maxenced** [comment]
Ok, this works as expected, at least for the tests I did : 

```puppet
  libvirt_pool { 'default' :
    ensure     => present,
    type       => 'logical',
    autostart  => true,
    sourcedev  => '/dev/sda5',
    sourcename => 'vm',
    target     => '/dev/vm',
  }
```

**Comment by maxenced** [comment]
@trefzer any opinion about this ?

File: templates/libvirtd.conf [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#
+# This file is managed with puppet (class ::libvirt::config)
+#    do not edit manually.
+#
+# Master configuration file for the QEMU driver.
+# All settings described here are optional - if omitted, sensible
+# defaults are used.
+#
+# See https://github.com/libvirt/libvirt/blob/master/src/qemu/qemu.conf
+# for possible values.
```
Comment (trefzer) [inline templates/libvirtd.conf]: copy paste, url is wrong

File: manifests/config.pp [hunk 2]
```diff
@@ -2,8 +2,9 @@
 #
 # Installs configuration files
 class libvirt::config (
-  $qemu_hook = $libvirt::qemu_hook,
-  $qemu_conf = $libvirt::qemu_conf,
+  $qemu_hook     = $libvirt::qemu_hook,
+  $qemu_conf     = $libvirt::qemu_conf,
+  $libvirtd_conf = $libvirt::qemu_conf,
```
Comment (trefzer) [inline manifests/config.pp]: wrong default, should not mix up witch qemu_conf

**Comment by maxenced** [comment]
Ah yeah sorry, mixed my branches ...

**Comment by maxenced** [comment]
Better now :)

**Comment by trefzer** [comment]
yes thanks. but can you also pull request the two other changes (in a separate pull request ?) so I can quickly merge them (I think they are almost good.).

So this one I dind not yet have the time to look at in detail.
So only short: can you provide some information about usage in the Readme ?
Did you write the spec tests, or are they from thias (you mentioned the source in the other files) ?

**Comment by maxenced** [comment]
Yes for the 2 other PR, just working on them right now. I wrote the spec file (which is quiet basic right now).

Ok for README

**Comment by maxenced** [comment]
Added readme

**Pull request merged**